benchmarks/benchmarks/torch_/scattering1d.py

```diff
@@ -0,0 +1,93 @@
+import torch
+from kymatio.torch import Scattering1D
+
+
+
+backends = []
+
+if torch.cuda.is_available():
+    devices = ['cuda', 'cpu']
+else:
+    devices = ['cpu']
+
+skcuda_available = False
+try:
+    if torch.cuda.is_available():
+        from skcuda import cublas
+        import cupy
+        skcuda_available = True
+except:
+    Warning('torch_skcuda backend not available.')
+
+if skcuda_available:
+    from kymatio.scattering1d.backend.torch_skcuda_backend import backend
+    backends.append(backend)
+    devices = ['cuda']
+
+from kymatio.scattering1d.backend.torch_backend import backend
+backends.append(backend)
+
+
+
+class BenchmarkScattering1D:
+    param_names = ["sc_params", "backend", "devices"]
+    timeout = 400  # in seconds
+
+    params = [
+        [
+            {  # Typical of EEG. J=8, Q=1, N=1024
+                # See Warrick et al. Physiological Measurement 2019
+                "J": 8,
+                "Q": 1,
+                "shape": 1024,
+                "batch_size": 32
+            },
+            {  # Typical of speech.
+                # See Andén and Mallat TASLP 2014
+                "J": 8,
+                "Q": 8,
+                "shape": 4096,
+                "batch_size": 32
+            },
+            {  # Typical of music.
+                # See Andén et al.
+                "J": 13,
+                "Q": 12,
+                "shape": 65536,
+                "batch_size": 32
+            },
+        ],
+        backends,
+        devices
+    ]
+    def setup(self, sc_params,  backend, device):
+        n_channels = 1
+        scattering = Scattering1D(backend=backend, J=sc_params["J"], shape=sc_params["shape"], Q=sc_params["Q"])
+        bs = sc_params["batch_size"]
+        if device == 'cuda':
+            bs *= 8
```

Comment:
I don't like this. Is there no way to include this in the parameters so it's obvious that the batch size is different? Alternatively, would it be so terrible if the CPU takes a long time? Maybe there's a compromise where the CPU is not too slow and the GPU is not too fast?

Reply:
yeah i've removed it. Yes it's terrible if the CPU takes long when you're developping